auto_builder.sh

```diff
@@ -114,10 +114,23 @@ case $value in
 
     2) ######################################################################################
         set -x
+        if [ ! -f $working_directory/sources/prometheus_client-$prometheus_client_version.tar.gz ]; then
+            wget -P $working_directory/sources/ https://github.com/prometheus/client_python/archive/v$prometheus_client_version.tar.gz
+            mv $working_directory/sources/v$prometheus_client_version.tar.gz $working_directory/sources/prometheus_client-$prometheus_client_version.tar.gz
```

Comment:
You can achieve this in one command with:
 wget ... -O $working_directory/sources/prometheus_client-$prometheus_client_version.tar.gz

Reply:
Good to know. I will update that at next update of the file. Thanks a lot 🙂